README.md

```diff
@@ -19,7 +19,7 @@ A handpicked list of __28808__ unique color names from [various sources](#source
 
 ## About 📋
 
-The aim of this project is to create as large a list of color names as possible while maintining a good name quality. We've merged various [lists](#sources-), modified the names when there were duplicates with different hex values, and shifted the colors a bit when there were identical colors with different names.
+The aim of this project is to create a list of color names as large as possible, while maintining a good name quality. We've merged various [lists](#sources-), modified the names when there were duplicates with different hex values, and shifted the colors a bit when there were identical colors with different names.
```

Comment:
I'd leave that as it was. It was written my a native speaker + I don't think it is confusing or wrong